WORKSPACE

```diff
@@ -33,11 +33,11 @@ bind(
 )
 
 # Determine SHA256 `wget https://github.com/envoyproxy/envoy-wasm/archive/$COMMIT.tar.gz && sha256sum $COMMIT.tar.gz`
-# envoy-wasm commit date: 08/26/2019
+# envoy-wasm commit date: 09/12/2019
 # bazel version: 0.28.1
-ENVOY_SHA = "ef4da0bda2bee932c575c0c98508c473dd6085c5"
+ENVOY_SHA = "20db8ed2938c9e88dd36605473b0019a23ab2657"
```

Comment:
Since this is pulling a new version of Envoy from upstream, could you also update `bazel version` here, and sync `.bazelversion` and `.bazelrc` with upstream?

Reply:
It builds fine with 0.28.1. Do we need to update bazel version?

Reply:
bazelrc diff is very large, and I'm not sure I can fix it correctly.

Reply:
Fine, I can update Bazel in a follow-up, but in general we try to keep Bazel in sync with whatever upstream uses.